planets.html

```diff
@@ -22,10 +20,8 @@
 	}
 	 else{
 		alert("Laaaaaammme.");
-	};
-
-	var guess, 
-		answer;
+	}; 
+		
 	guess = prompt("This one should be easy.");
```

Comment:
the structure prompt-answer-if gets repeated. a function that takes a few strings would remove that repetition.